core/src/main/java/org/mule/runtime/core/internal/processor/AsyncDelegateMessageProcessor.java

```diff
@@ -192,15 +198,14 @@ public void start() throws MuleException {
   public void stop() throws MuleException {
     super.stop();
     stopIfNeeded(delegate);
-
+    stopIfNeeded(backpressureHandler);
```

Comment:
this should be wrapped in FunctionalUtils.safely() so that if this fails, the rest is disposed correctly